lint-staged.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 	],
 	"package.json": (files) => [
 		"yarn constraints --fix",
+		"yarn",
```

Comment:
```suggestion
		"yarn constraints --fix && yarn install",
```

Nice, one suggestion - since it's connected to the constraints command, let's chain them maybe and be explicit with the yarn command we're running since `yarn` is an alias for `yarn install`. Thanks for tackling this.

Reply:
Great idea! Just committed that. Definitely a fun issue to dig into. ✨